### PR TITLE
ROC-2719 Add Sole trader to defendant

### PR DESCRIPTION
**Pull request by robertparkinson**
https://tools.hmcts.net/jira/browse/ROC-2719

### CODE REVIEW

File: src/test/app/forms/models/claimantAddition.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { Validator } from 'class-validator'
 
 import { expectValidationError } from './validationUtils'
 
-import { PartyTypes as DefendantTypes } from 'forms/models/partyTypes'
+import { PartyType as DefendantTypes } from '../../../../main/app/common/partyType'
```
Comment (dharmendrak) [inline src/test/app/forms/models/claimantAddition.ts]: import { PartyType  as DefendantTypes} from 'app/common/partyType'

File: src/test/app/claims/model/yours/party.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 import { expect } from 'chai'
 import { Address } from 'app/claims/models/address'
-import { PartyTypes } from 'forms/models/partyTypes'
+import { PartyType } from '../../../../../main/app/common/partyType'
```
Comment (dharmendrak) [inline src/test/app/claims/model/yours/party.ts]: import { PartyType } from 'app/common/partyType'

File: src/test/app/claims/model/yours/organisation.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 import { expect } from 'chai'
 import { Address } from 'app/claims/models/address'
-import { PartyTypes } from 'forms/models/partyTypes'
+import { PartyType } from '../../../../../main/app/common/partyType'
```
Comment (dharmendrak) [inline src/test/app/claims/model/yours/organisation.ts]: import { PartyType } from 'app/common/partyType'

File: src/test/app/claims/model/yours/individual.ts [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
 import { expect } from 'chai'
 import { Address } from 'app/claims/models/address'
-import { PartyTypes } from 'forms/models/partyTypes'
+import { PartyType } from '../../../../../main/app/common/partyType'
```
Comment (dharmendrak) [inline src/test/app/claims/model/yours/individual.ts]: import { PartyType } from 'app/common/partyType'

File: src/test/app/claims/model/theirs/theirDetails.ts [hunk 5]
```diff
@@ -3,7 +3,7 @@
 
 import { expect } from 'chai'
 import { Address } from 'app/claims/models/address'
-import { PartyTypes } from 'forms/models/partyTypes'
+import { PartyType } from '../../../../../main/app/common/partyType'
```
Comment (dharmendrak) [inline src/test/app/claims/model/theirs/theirDetails.ts]: import { PartyType } from 'app/common/partyType'

File: src/test/app/claims/model/theirs/organisation.ts [hunk 6]
```diff
@@ -3,7 +3,7 @@
 
 import { expect } from 'chai'
 import { Address } from 'app/claims/models/address'
-import { PartyTypes } from 'forms/models/partyTypes'
+import { PartyType } from '../../../../../main/app/common/partyType'
```
Comment (dharmendrak) [inline src/test/app/claims/model/theirs/organisation.ts]: import { PartyType } from 'app/common/partyType'

File: src/test/app/claims/model/theirs/individual.ts [hunk 7]
```diff
@@ -4,7 +4,7 @@
 import { expect } from 'chai'
 import { Address } from 'app/claims/models/address'
 import { Individual } from 'claims/models/theirs/individual'
-import { PartyTypes } from 'forms/models/partyTypes'
+import { PartyType } from '../../../../../main/app/common/partyType'
```
Comment (dharmendrak) [inline src/test/app/claims/model/theirs/individual.ts]: import { PartyType } from 'app/common/partyType'

File: src/test/app/forms/models/claimantDetails.ts [hunk 8]
```diff
@@ -6,7 +6,7 @@ import { Validator } from 'class-validator'
 import { expectValidationError } from './validationUtils'
 
 import { ValidationErrors as CommonValidationErrors } from 'forms/validation/validationErrors'
-import { PartyTypes as ClaimantTypes } from 'forms/models/partyTypes'
+import { PartyType as ClaimantTypes } from '../../../../main/app/common/partyType'
```
Comment (dharmendrak) [inline src/test/app/forms/models/claimantDetails.ts]: import { PartyType  as ClaimantTypes} from 'app/common/partyType'

File: src/test/app/forms/models/defendantAddition.ts [hunk 9]
```diff
@@ -5,7 +5,7 @@ import { Validator } from 'class-validator'
 
 import { expectValidationError } from './validationUtils'
 
-import { PartyTypes as DefendantTypes } from 'forms/models/partyTypes'
+import { PartyType as DefendantTypes } from '../../../../main/app/common/partyType'
```
Comment (dharmendrak) [inline src/test/app/forms/models/defendantAddition.ts]: import { PartyType as DefendantTypes } from 'app/common/partyType'

File: src/test/app/forms/models/defendantDetails.ts [hunk 10]
```diff
@@ -5,7 +5,7 @@ import { Validator } from 'class-validator'
 
 import { expectValidationError } from './validationUtils'
 
-import { PartyTypes as DefendantTypes } from 'forms/models/partyTypes'
+import { PartyType as DefendantTypes } from '../../../../main/app/common/partyType'
```
Comment (dharmendrak) [inline src/test/app/forms/models/defendantDetails.ts]: import { PartyType as DefendantTypes } from 'app/common/partyType'

File: src/test/data/draft/partyDetails.ts [hunk 11]
```diff
@@ -1,9 +1,8 @@
-import { PartyTypes } from 'forms/models/partyTypes'
+import { PartyType } from '../../../main/app/common/partyType'
```
Comment (dharmendrak) [inline src/test/data/draft/partyDetails.ts]: import { PartyType } from 'app/common/partyType'

File: src/test/data/entity/party.ts [hunk 12]
```diff
@@ -1,9 +1,8 @@
-import { PartyTypes } from 'forms/models/partyTypes'
+import { PartyType } from '../../../main/app/common/partyType'
```
Comment (dharmendrak) [inline src/test/data/entity/party.ts]: import { PartyType as DefendantTypes } from 'app/common/partyType'

File: src/test/http-mocks/claim-store.ts [hunk 13]
```diff
@@ -1,7 +1,7 @@
 import * as config from 'config'
 import * as mock from 'nock'
 import * as HttpStatus from 'http-status-codes'
-import { PartyTypes } from 'forms/models/partyTypes'
+import { PartyType } from '../../main/app/common/partyType'
```
Comment (dharmendrak) [inline src/test/http-mocks/claim-store.ts]: import { PartyType  } from 'app/common/partyType'

File: src/test/app/claims/model/theirs/individual.ts (hunk 7 above)
Comment (robertparkinson) [reply src/test/app/claims/model/theirs/individual.ts]: done

File: src/main/app/forms/models/claimantDetails.ts [hunk 14]
```diff
@@ -1,6 +1,6 @@
 import { Serializable } from 'app/models/serializable'
 import { IsDefined, IsIn, IsOptional, MaxLength, ValidateIf } from 'class-validator'
-import { PartyTypes, PartyTypes as ClaimantTypes } from 'app/forms/models/partyTypes'
+import { PartyType, PartyType as ClaimantTypes } from 'app/common/partyType'
```
Comment (sabahirfan) [inline src/main/app/forms/models/claimantDetails.ts]: Please use either the  `PartyType` or the alias `ClaimantTypes`. Prefer to name alias as singular `ClaimantType`.

File: src/main/app/forms/models/claimantDetails.ts [hunk 15]
```diff
@@ -69,6 +69,6 @@ export class ClaimantDetails implements Serializable<ClaimantDetails> {
   }
 
   toString (): string {
-    return this.type.value === PartyTypes.INDIVIDUAL.value ? this.fullName : this.organisation
+    return this.type.value === PartyType.INDIVIDUAL.value ? this.fullName : this.organisation
```
Comment (sabahirfan) [inline src/main/app/forms/models/claimantDetails.ts]: please use alias `ClaimantType` to be consistent.

File: src/main/app/forms/models/defendantDetails.ts [hunk 16]
```diff
@@ -1,6 +1,6 @@
 import { Serializable } from 'app/models/serializable'
 import { IsDefined, IsIn, IsOptional, MaxLength, ValidateIf } from 'class-validator'
-import { PartyTypes as DefendantTypes } from 'app/forms/models/partyTypes'
+import { PartyType as DefendantTypes } from 'app/common/partyType'
```
Comment (sabahirfan) [inline src/main/app/forms/models/defendantDetails.ts]: I will prefer alias singular `DefendantType`

**Pull request merged**